cmd/neofs-adm/internal/modules/morph/root.go

```diff
@@ -87,4 +87,5 @@ func init() {
 	RootCmd.AddCommand(forceNewEpoch)
 	forceNewEpoch.Flags().String(alphabetWalletsFlag, "", "path to alphabet wallets dir")
 	forceNewEpoch.Flags().StringP(endpointFlag, "r", "", "N3 RPC node endpoint")
+	forceNewEpoch.Flags().String(contractsInitFlag, "", "path to archive with compiled NeoFS contracts")
```

Comment:
I suggest to reconsider this flag. NNS should be enough to find contract script hashes, isn't it?